src/main/java/com/fasterxml/jackson/databind/InjectableValues.java

```diff
@@ -76,7 +79,13 @@ public Object findInjectableValue(Object valueId, DeserializationContext ctxt,
             String key = (String) valueId;
             Object ob = _values.get(key);
             if (ob == null && !_values.containsKey(key)) {
-                throw new IllegalArgumentException("No injectable id with value '"+key+"' found (for property '"+forProperty.getName()+"')");
+                final JacksonInject.Value injectableValue = ctxt.getAnnotationIntrospector()
```

Comment:
Same here too; really should handle introspection so we would not need to do that processing every time injectable value not found. Not sure if that is doable.